app/Http/Controllers/Apis/Protected/Summit/Factories/SummitSponsorshipAddOnTypeValidationRulesFactory.php

```
@@ -0,0 +1,36 @@
+ * @package App\Http\Controllers
+ */
+final class SummitSponsorshipAddOnTypeValidationRulesFactory extends AbstractValidationRulesFactory
+{
+    public static function buildForAdd(array $payload = []): array
+    {
+        return [
+            'name' => 'required|string|max:255',
+        ];
+    }
+
+    public static function buildForUpdate(array $payload = []): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$payload'. (undefined)

(UnusedFormalParameter)